torch/_functorch/_aot_autograd/subclass_parametrization.py

```diff
@@ -15,6 +16,8 @@ class SubclassCreationMeta:
     class_type: Any
     attrs: dict[str, "SubclassCreationMeta"]
     metadata: Any
+    outer_size: Iterable[Union[None, int, torch.SymInt]]
+    outer_stride: Iterable[Union[None, int, torch.SymInt]]
```

Comment:
this looks more correct, but did you actually need it to get some tests to pass?

Reply:
Yup I needed this for passing a test, not sure why it never got triggered in parameters case.